### DISCUSSION TITLE
Feature request: add camera's metadata to EXIF

### DISCUSSION THREAD

**Discussion by nanawel** [Motion-Project/motion#1510] [General]
0. Reviewed guide and contributing documents? (Yes/No):  **Yes**
1. version [x.y.z, hash, other]:  **4.3.2**
2. installed as a package or compiled from sources [deb, rpm, git, other]:  **package (Arch)**
3. standalone or part of third party [motion, MotionEyeOS, other]:  **standalone**
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]:  **mmal**
5. hardware [x86, ARM, other]:  **ARM**
6. operating system [16.04, Stretch, etc, FreeBSD, other]:  **ArchlinuxARM**

Hi,

Would it be possible to save camera's metadata such as `ExposureTime`, `ShutterSpeedValue`, `ExposureMode`, etc. in each picture?
That doesn't seem to be a limitation of the MMAL source since I don't see much metadata in pictures taken with a V4L2 camera with motion either.

Sorry if this feature as already been discussed, I could not find any trace of it.

**Comment by nanawel** [comment]
I see the original issue has been closed. Any update on this enhancement?

**Reply by Mr-Dave** [reply]
Couple of comments:
1.  All enhancements associated with Motion have been closed as discussed in the [poll](https://github.com/Motion-Project/motion/discussions/1794).
2. MMAL is no longer used upstream and therefore was removed from Motion. The libcamerify option that replaces it and provides the interface between the Pi camera and C programs does not provide any controls.   As a result, none of the meta items indicated could be added in Motion even if desired.

**Reply by nanawel** [reply]
Thanks, I missed this info. It all seems clear to me now :ok_hand:
